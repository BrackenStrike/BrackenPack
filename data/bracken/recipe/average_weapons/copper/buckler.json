{
	"type": "minecraft:smithing_transform",
	"template": "minecraft:vex_armor_trim_smithing_template",
	"base": "minecraft:stone_sword",
	"addition": "minecraft:copper_ingot",
	"result": {
		"id": "minecraft:stone_sword",
		"components": {
			"minecraft:max_damage": 175,
			"minecraft:item_name": "Copper Buckler",
			"minecraft:item_model": "bracken:average_weapons/copper/buckler",
			"minecraft:lore":[ 
			{"translate":"","color":"white","italic":false},
			{"translate":"When in Main Hand:","color":"gray","italic":false},
			{"translate": "3.5 Attack Damage","color":"dark_green","italic":false},
			{"translate": "1.6 Attack Speed","color":"dark_green","italic":false},
			{"translate": "+3 Armor","color":"dark_green","italic":false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:copper_buckler",
					"amount": 2.5,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:copper_buckler",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:armor",
					"id": "bracken:copper_buckler",
					"amount": 3,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}