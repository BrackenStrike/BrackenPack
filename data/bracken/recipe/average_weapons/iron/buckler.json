{
	"type": "minecraft:smithing_transform",
	"template": "minecraft:vex_armor_trim_smithing_template",
	"base": "minecraft:iron_sword",
	"addition": "minecraft:iron_ingot",
	"result": {
		"id": "minecraft:iron_sword",
		"components": {
			"minecraft:item_name": "Iron Buckler",
			"minecraft:item_model": "bracken:average_weapons/iron/buckler",
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "4 Attack Damage","color":"dark_green","italic":false},
				{"translate": "1.6 Attack Speed","color":"dark_green","italic":false},
				{"translate": "+3 Armor","color":"dark_green","italic":false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:iron_buckler",
					"amount": 3,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:iron_buckler",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:armor",
					"id": "bracken:iron_buckler",
					"amount": 3,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}