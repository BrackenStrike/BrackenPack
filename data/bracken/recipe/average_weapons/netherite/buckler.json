{
	"type": "minecraft:smithing_transform",
	"template": "minecraft:vex_armor_trim_smithing_template",
	"base": "minecraft:netherite_sword",
	"addition": "minecraft:netherite_ingot",
	"result": {
		"id": "minecraft:netherite_sword",
		"components": {
			"minecraft:item_name": "Netherite Buckler",
			"minecraft:item_model": "bracken:average_weapons/netherite/buckler",
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "6 Attack Damage","color":"dark_green","italic":false},
				{"translate": "1.6 Attack Speed","color":"dark_green","italic":false},
				{"translate": "+4 Armor","color":"dark_green","italic":false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:netherite_buckler",
					"amount": 5,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:netherite_buckler",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:armor",
					"id": "bracken:netherite_buckler",
					"amount": 4,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}