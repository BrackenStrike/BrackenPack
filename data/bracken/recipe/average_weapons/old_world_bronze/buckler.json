{
	"type": "minecraft:smithing_transform",
	"template": "minecraft:vex_armor_trim_smithing_template",
	"base": "minecraft:netherite_sword",
	"addition": "minecraft:cat_spawn_egg",
	"result": {
		"id": "minecraft:netherite_sword",
		"components": {
			"minecraft:item_name": "\"Old World Bronze Buckler\"",
			"minecraft:item_model": "bracken:average_weapons/old_world_bronze/buckler",
			"minecraft:lore": [
				"{\"translate\":\"\",\"color\":white,\"italic\":false}",
				"{\"translate\":\"When in Main Hand:\",\"color\":gray,\"italic\":false}",
				"{\"translate\":\" 6.5 Attack Damage\",\"color\":dark_green,\"italic\":false}",
				"{\"translate\":\" 1.6 Attack Speed\",\"color\":dark_green,\"italic\":false}",
				"{\"translate\":\" +4 Armor\",\"color\":dark_green,\"italic\":false}"
			],
			"minecraft:attribute_modifiers": {
				"modifiers": [{
					"type": "minecraft:attack_damage",
					"id": "bracken:netherite_buckler",
					"amount": 5.5,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:netherite_buckler",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:armor",
					"id": "bracken:netherite_buckler",
					"amount": 4,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"show_in_tooltip": false
			}
		}
	}
}