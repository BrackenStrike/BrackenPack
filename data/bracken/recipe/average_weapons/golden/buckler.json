{
	"type": "minecraft:smithing_transform",
	"template": "minecraft:vex_armor_trim_smithing_template",
	"base": "minecraft:golden_sword",
	"addition": "minecraft:gold_ingot",
	"result": {
		"id": "minecraft:golden_sword",
		"components": {
			"minecraft:item_name": "Golden Buckler",
			"minecraft:item_model": "bracken:average_weapons/golden/buckler",
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "2 Attack Damage","color":"dark_green","italic":false},
				{"translate": "1.6 Attack Speed","color":"dark_green","italic":false},
				{"translate": "+3 Armor","color":"dark_green","italic":false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:golden_buckler",
					"amount": 1,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:golden_buckler",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:armor",
					"id": "bracken:golden_buckler",
					"amount": 3,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}